portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Properties.testcase

```diff
@@ -86,12 +86,25 @@ definition {
 	}
 
 	@description = "Feature ID: Legacy | Automation ID: LRAC-8159 | Test Summary: The properties list in settings shows a list of all properties in the workspace"
-	@ignore = "true"
 	@priority = "5"
 	test PropertiesListInSettingsShowsListOfAllPropertiesInTheWorkspace {
+		var propertyName = ACDXPSettings.connectDXPtoAnalyticsCloud();
 
-		// TODO LRAC-8159 PropertiesListInSettingsShowsListOfAllPropertiesInTheWorkspace pending implementation
+		ACUtils.launchAC();
+
+		ACNavigation.goToSettings();
+
+		Click(
+			key_SettingsNavigation = "properties",
+			locator1 = "ACSettings#SETTINGS_NAVIGATION");
+
+		AssertElementPresent(
+			itemName = "${propertyName}",
+			locator1 = "ACSettings#ITEM_ON_LIST");
```

Comment:
Hi @91nilo. For these assertions I created a macro so it should be easier to read. It would look something like this: `ACProperties.assertNewCreatedProperty(propertyName = "Name of property");`

Reply:
Hey Josh, i created another macro (assertPropertySettingsList) because the assertNewCreatedProperty doesn't does the assert on the settings list, so i thought it could not be applied on this test, do you think i did right or i should use the assertNewCreatedProperty?

Reply:
Oh I see what you mean. Yea my macro is used more for single property assertions. The one you created will check the entire list which is what you're looking for. Look good!